tests/gemini/test_gemini_autolog.py

```diff
@@ -4,16 +4,23 @@
 """
 
 import base64
+import importlib.metadata
+import os
 from unittest.mock import patch
 
 import pytest
 from google import genai
+from packaging.version import Version
 
 import mlflow
 from mlflow.entities.span import SpanType
 
 from tests.tracing.helper import get_traces
 
+is_gemini_larger_than_1_5 = Version(importlib.metadata.version("google.genai")) > Version(
+    "1.5.0"
+) or os.environ.get("GOOGLE_GENAI_DEV")
```

Comment:
```suggestion
) or os.environ.get("GOOGLE_GENAI_DEV") == 'true'
```

Reply:
btw why do we need this environment variable?

Reply:
The master branch of the repository still uses 1.5.0 in pyproject.toml

Reply:
Oh, actually they released 1.7.0 today